#! /bin/sh /usr/share/dpatch/dpatch-run
## 11-autoconf_ACE_LACKS_NAMED_POSIX_SEM_fix.dpatch
##
## DP: do not fail if the sem_open test is run twice

@DPATCH@
--- ACE_wrappers.orig/configure.ac
+++ ACE_wrappers/configure.ac
@@ -4313,6 +4313,9 @@
   if (sem_close (s) != 0)
     return -1;  /* Something went wrong! */
 
+  /* ignore error here */
+  sem_unlink("ace_semaphore_foo");
+
   return 0;
 }
              ]])],[
